composer.json

```diff
@@ -113,7 +113,10 @@
     "autoload-dev": {
         "psr-4": {
             "ApiPlatform\\Core\\Tests\\": "tests/"
-        }
+        },
+        "classmap": [
+            "vendor/phpunit/phpunit/tests/"
```

Comment:
What is this for? :thinking:

Reply:
It’s mandatory to test the assertion I imported from PHPUnit itself

Reply:
https://github.com/api-platform/core/pull/2938